From c3c53e4bcd8026b7fd46203322acb4cabf6b459c Mon Sep 17 00:00:00 2001
From: Augusto Caringi <acaringi@redhat.com>
Date: Thu, 22 Nov 2018 15:55:30 +0100
Subject: [PATCH] Install *_example.txt files to tools/doc (they are referenced
 in man pages)

---
 tools/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/tools/CMakeLists.txt b/tools/CMakeLists.txt
index 10002a7..e25d858 100644
--- a/tools/CMakeLists.txt
+++ b/tools/CMakeLists.txt
@@ -1,2 +1,5 @@
 file(GLOB BT_FILES *.bt)
+file(GLOB TXT_FILES *.txt)
+list(REMOVE_ITEM TXT_FILES ${CMAKE_CURRENT_SOURCE_DIR}/CMakeLists.txt)
 install(FILES ${BT_FILES} DESTINATION share/bpftrace/tools)
+install(FILES ${TXT_FILES} DESTINATION share/bpftrace/tools/doc)
-- 
2.17.2

